### ISSUE TITLE
urlopen behaves differently on python3

### ISSUE THREAD

**Issue by Grazfather** [hugsy/gef#102]
#### Steps to reproduce

  1. Open gdb on a machine where it's compiled with python3
  1. `shellcode get 23`
  1. See 'raw' python error

#### Observed Results

  * exception
#### Expected results

  * error message

On python2 gdb
```python
GEF for darwin ready, type `gef' to start, `gef config' to configure
49 commands loaded for GDB 7.12 using Python engine 2.7
[*] 4 commands could not be loaded, run `gef missing` to know why.
gef> shellcode get 23
[!] Could not query search page: got 404
gef>
```

On python3 gdb
```python
GEF for linux ready, type `gef' to start, `gef config' to configure
52 commands loaded for GDB 7.7.1 using Python engine 3.4
[*] 1 commands could not be loaded, run `gef missing` to know why.
Reading symbols from a.out...(no debugging symbols found)...done.
gef➤  shellcode get 23
Python Exception <class 'urllib.error.HTTPError'> HTTP Error 404: Not Found:
Error occurred in Python command: HTTP Error 404: Not Found
```

This can be handled a few ways. The cleanest is probably to write a wrapper once that catches the exception on 3 and wraps it with something containing the error code (instead of duplicating this code every time it's called).

**Comment by hugsy** [comment]
Should be fixed by 317d962

Python 3
```
gef➤  shellcode get 1337
[!] Failed to fetch shellcode #1337
```

Python 2
```
gef> shellcode get 1337
[!] Failed to fetch shellcode #1337
```

**Comment by Grazfather** [comment]
Looks good to me!
